src/test/java/com/wheretoeat/wheretoeat/RestaurantDTOTests.java

```diff
@@ -0,0 +1,55 @@
+package com.wheretoeat.wheretoeat;
```

Comment:
This is just a unit test example, you can continue this format with other DTO. Or you could use that `@Before` annotation.